.travis.yml

```diff
@@ -28,14 +27,14 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower
+  - npm install -g bower yarn
```

Comment:
yarn team recommends that yarn is *not* installed via npm; that said, i am currently installing via npm at work because it was hard to install OS package (ubuntu in CI). Not sure if travis makes this easier.